README.md

```diff
@@ -29,7 +29,7 @@ To update the CLI to a new version, simply download a new release and replace th
 
 ### NPM install
 
-If you are a Node.js developer, you may wish to install the balena CLI through npm[https://www.npmjs.com]. The npm installation involves building native (platform-specific) binary modules, for which there are some pre-requisites:
+If you are a Node.js developer, you may wish to install the balena CLI through (npm)[https://www.npmjs.com]. The npm installation involves building native (platform-specific) binary modules, for which there are some pre-requisites:
```

Comment:
* Swapped plain brackets and square brackets (click the "View file" button for a preview of the change.)
* By the way, I am already making this change in this other PR: https://github.com/balena-io/balena-cli/pull/1175 :-)

Reply:
Great minds! Thanks for pointing that out :)